templates/environment/cf.yml

```
@@ -216,7 +167,7 @@ Resources:
     Properties:
       Scheme: internet-facing
       SecurityGroups: [ !GetAtt PublicLoadBalancerSecurityGroup.GroupId ]
-      Subnets: [ !Ref PublicSubnet1, !Ref PublicSubnet2 ]
+      Subnets: [ {{if .ImportVpc}}{{range $id := .ImportVpc.PublicSubnetIDs}}{{$id}}, {{end}}{{else}}{{range $ind, $cidr := .VpcConfig.PublicSubnetCIDRs}}!Ref PublicSubnet{{inc $ind}}, {{end}}{{end}}]
```

to keep these slightly more readable what do you think of:
```suggestion
      {{- if .ImportVPC}}
      Subnets: [ {strjoin .ImportVpc.PublicSubnetIDs ","}}
      {{- else }}
      Subnets: [{{range $ind, $cidr := .VpcConfig.PublicSubnetCIDRs}}!Ref PublicSubnet{{inc $ind}}, {{end}}]
      {{- end}}
```